src/wallet/rpcwallet.cpp

```diff
@@ -2413,11 +2417,17 @@ UniValue listunspent(const JSONRPCRequest& request)
         }
     }
 
+    bool include_untrusted = true;
+    if (request.params.size() > 3 && !request.params[3].isNull()) {
+        RPCTypeCheckArgument(request.params[3], UniValue::VBOOL);
+        include_untrusted = request.params[3].get_bool();
+    }
+
     UniValue results(UniValue::VARR);
     vector<COutput> vecOutputs;
     assert(pwalletMain != NULL);
     LOCK2(cs_main, pwalletMain->cs_wallet);
-    pwalletMain->AvailableCoins(vecOutputs, false, NULL, true);
+    pwalletMain->AvailableCoins(vecOutputs, !include_untrusted, NULL, true);
```

Comment:
note: in the future we should rename and invert the AvailableCoins argument to this.